src/pages/Dashboard/Portfolio/index.tsx

```diff
@@ -23,25 +23,14 @@ export default function PortfolioWidget() {
   //   setSelectChain(newChain)
   // }
   const { data: balances, isLoading } = useGetChainsBalances()
-  const [availableBalances, setAvailableBalances] = useState<{ chainID: ChainsId; balance: number }[]>([])
+  const [availableBalances, setAvailableBalances] = useState<{ chainID: number; balance: number }[]>([])
+
+  ALL_CHAINS.push(AllChain)
```

Comment:
should not directly push to imported array. instead define cloned array and push to that array

Reply:
This modification has been done by pedro. How would you like to proceed @prodesert22 ?

Reply:
Updated by @jb1011 ✅